Feature branch

Rayan84 opened the pull request:
This project is a collaborative effort to achieve replica of Newsweek magazine https://www.newsweek.com/.
Technologies Used:

- HTML
- CSS
- Bootstrap
- JavaScript

Lichapa committed:
Add Navbar

Rayan84 committed:
Add weather Icon, city name and temprature
Add style to the navbar
Add style to body
Add main and contents

Lichapa committed:
Add images on more stories section

Lichapa committed:
Add footer

Lichapa committed:
Add In-Magazine
Add Editor's pick

Lichapa committed:
Fix IN-Magazine img
Add w-100

Rayan84 committed:
Add HTML content sections:

Lichapa committed:
Update README.md

Lichapa committed:
Added screenshot to README.md

Lichapa committed:
Add Mobile Nav bar

Rayan84 committed:
Add fixed position to the nav
Add JS for fixed nav when scroll down

Lichapa committed:
Fix CSS lint errors

Lichapa committed:
Fix HTML validation errors

Lichapa committed:
Fix footer Socials
Add class footer social in media querry

javitocor reviewed (changes requested):
# STATUS: CHANGES REQUIRED ♻️ 

Great job so far with this project, team! 👍 
And with only using 141 lines of CSS, amazing! 👏 
Pretty impressive your use of JS in these project! ⭐ 
I can see you really know how to use bootstrap, and very well!!! 💯 
Congratulations!!! 🎂 

However, you need to do some changes before the project gets the approval! but seeing how you use bootstrap it will not take you too long!
Please, check my comments below 👇 
And once the required changes are complete resubmit your code for a new code review

Good luck with the changes, team, and keep the good work up 😃 

You can reach me out in Slack if you have any question or doubt.

## CHANGES

-General:
   - In features and top story the topic must be included inside the image, same in the magazine and editors pick:
![Capture1](https://user-images.githubusercontent.com/56809963/94042328-b9771900-fdcb-11ea-8df4-7a8297a881ec.PNG)
![Capture](https://user-images.githubusercontent.com/56809963/94042337-bbd97300-fdcb-11ea-891b-44caa15b3fba.PNG)
   - One small section is missing and it is important for smaller screens
![Capture2](https://user-images.githubusercontent.com/56809963/94042447-e3c8d680-fdcb-11ea-8685-91c35db15bcc.PNG)
   - In the magazine section, the first image is different from the rest. Kindly add it
![Capture3](https://user-images.githubusercontent.com/56809963/94042569-0e1a9400-fdcc-11ea-9d7a-0ef3050c2cb1.PNG)
   - My turn section and culture& travel sections are also missing, please add it to your project
  - In the opinion section many opinions are missing
  - In the original website the first breakdown is at 1000px, your is at 1200px. Please, keep the breakdowns as in the original site.
   - The navbar should fit the 100% of the screen always. Please fix it.
![Capture5](https://user-images.githubusercontent.com/56809963/94043093-a9136e00-fdcc-11ea-9123-07953b31d67b.PNG)
![Capture4](https://user-images.githubusercontent.com/56809963/94043095-aa449b00-fdcc-11ea-96eb-64440a96838c.PNG)
   - Please fix the images in the latest sections, they are overlapping
- Breakpoint 996px
   - Your navbar does not let to see the title of the top story and features stories sections, please fix it.
![Capture6](https://user-images.githubusercontent.com/56809963/94043290-fee81600-fdcc-11ea-9838-c2cef05f0474.PNG)
   - The in the magazine and the editors pick sections are not following the layout of the original page, please fix it
![Capture8](https://user-images.githubusercontent.com/56809963/94043681-8766b680-fdcd-11ea-975f-d483c370f05e.PNG)
![Capture7](https://user-images.githubusercontent.com/56809963/94043697-89c91080-fdcd-11ea-95a4-44fc5bcf9eee.PNG)
   - Us, culture, sports, world, business, tech and science sections are not following the original layout neither. Please modify it
![Capture10](https://user-images.githubusercontent.com/56809963/94043990-dc0a3180-fdcd-11ea-89c0-5dd5e2be7eb5.PNG)
![Capture9](https://user-images.githubusercontent.com/56809963/94044000-de6c8b80-fdcd-11ea-86ee-cda4182d5c31.PNG)
   - Please adjust the size of the columns in the newsletter section
![Capture12](https://user-images.githubusercontent.com/56809963/94044151-0eb42a00-fdce-11ea-844b-16637f9b6aaa.PNG)
![Capture11](https://user-images.githubusercontent.com/56809963/94044160-107ded80-fdce-11ea-8212-c9bfd35205bc.PNG)
- Breakpoint 768px
  - The order of the sections should be: top story, featured stories, opinion, more stories, my turn, culture and travel. Kindly make yours in the same order.
   - Please fix the width of the world, business, sport, tech&science.... sections, also the layout must be fixed.
![Capture14](https://user-images.githubusercontent.com/56809963/94044612-aca7f480-fdce-11ea-8433-c7b0865026d2.PNG)
![Capture13](https://user-images.githubusercontent.com/56809963/94044622-ae71b800-fdce-11ea-9fe0-b0c3e10331d4.PNG)

   - The image in the newsletter section should disappear, please remove your for this screen size.
- Please also fix the margin of the section for the small devices, adjust it to the original page.

Lichapa committed:
Fix Features and Top Story section to include topic inside image
Add A signup segment in Opinion section
Add My Turn and Culture section
Fix Navbar to be fit 100% on all Screen sizes
Fix the breakdonwn points
Add Margin to top story to show heading at 996px
Fix layout of Editor's pick, tech and scice, in the margazin sectionk at 996px
Adjust columns of Newsletter @996px
Made changes for 764px screen

Lichapa committed:
fix lint error

umairarshadbutt commented on index.html:
```diff
@@ -15,11 +15,1765 @@
+                <span class="newsletter-img"></span>
+              </div>
+              <article class="col col-md-7 col-sm-10 m-auto py-3">
+                <h5>The Starting 5</h5>
```
Align center this heading

umairarshadbutt commented on assets/css/styles.css:
```diff
@@ -0,0 +1,147 @@
+  }
+
+  .footer-social {
+    text-align: left !important;
```
Remove `!imortant`

umairarshadbutt commented on index.html:
```diff
@@ -15,11 +15,1765 @@
+                </div>
+                <div class="col-4">
+                  <img
+                    style="width: 4rem"
```
Remove inline CSS

umairarshadbutt commented on index.html:
```diff
@@ -15,11 +15,1765 @@
+
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Remove inline CSS

umairarshadbutt commented:
##
## Pull Request under review by Umair Arshad (TSE)

umairarshadbutt commented on index.html:
```diff
@@ -15,11 +15,1765 @@
+
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Remove inline CSS

umairarshadbutt commented on index.html:
```diff
@@ -15,11 +15,1765 @@
+
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Remove inline CSS

umairarshadbutt commented on index.html:
```diff
@@ -15,11 +15,1765 @@
+            <ul class="list-unstyled">
+              <li class="media py-3 border-bottom border-bottom-double">
+                <img
+                  style="width: 4rem"
```
Remove inline CSS

umairarshadbutt commented on index.html:
```diff
@@ -15,11 +15,1765 @@
+              <li class="pt-3"><strong>Opinion</strong></li>
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Remove inline CSS

umairarshadbutt reviewed (changes requested):
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly make sure the Last reviewer's changes and work on the suggested changes 👇.
- [x] Add heading of **My turn** section and **culture& travel** sections 
- [x] Please adjust the size of the columns in the newsletter section
- [x] Remove inline CSS and `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)




##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

Lichapa committed:
fix lint error

Lichapa committed:
Removed important in css
Add text-sm-left, add text-md-right to socials icons to align left or right on different breakpoints

Lichapa committed:
Removed Inline on opinion IMG CSS

Lichapa committed:
Add heading of My turn section and culture& travel sections

umairarshadbutt commented:
##
## Pull Request under review by Umair Arshad (TSE)

umairarshadbutt reviewed (changes requested):
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Rayan84 and @Lichapa
**Good job with the changes! :tada: We are almost there.**

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

- [ ] Manage the sections order like the original landing page on small screens

### Navbar
- [x] There is a still issue in the navbar on medium and small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://lichapa.github.io/Newsweek-clone/)|
|:---:|:---:|
|![source-page](https://user-images.githubusercontent.com/29689273/94132806-4035ff80-fe79-11ea-8fb8-086afe47cfca.png)|![clone](https://user-images.githubusercontent.com/29689273/94132843-4d52ee80-fe79-11ea-97bc-f6431db3adbc.png)|

### Subscribe
- [x] Text should be overlapped on the medium and small screen

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://lichapa.github.io/Newsweek-clone/)|
|:---:|:---:|
|![source-page](https://user-images.githubusercontent.com/29689273/94133543-39f45300-fe7a-11ea-9e84-e52d592f2878.png)|![clone](https://user-images.githubusercontent.com/29689273/94133555-3d87da00-fe7a-11ea-919a-bd59935d4985.png)|

### Footer
- [x] remove `mt` from social media icons (on small screens)

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://lichapa.github.io/Newsweek-clone/)|
|:---:|:---:|
|![source-page](https://user-images.githubusercontent.com/29689273/94133698-72942c80-fe7a-11ea-8c39-b3a1950f0db0.png)|![clone](https://user-images.githubusercontent.com/29689273/94133709-74f68680-fe7a-11ea-901f-4d57c60dc0ef.png)|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

Lichapa committed:
Fix Navbar on Medium and Small screens

Lichapa committed:
Fix footer social icon  paddin-top on small screen

Rayan84 committed:
add style to nav-2:
center elements in subscribe section in sm size
give position absolute for images inside subscribe section in sm size
give size auto for subscribe button

Rayan84 committed:
change order

Lichapa committed:
Fix stylelint errors

Lichapa committed:
Fix html validation errors

Lichapa committed:
Fix order of sections to match the original page

arslanbisharat reviewed (approved):
### Congratulations APPROVED :clap:
Hello, I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat
